Source/WebCore/rendering/svg/RenderSVGInlineText.cpp

```diff
@@ -209,16 +209,19 @@ void RenderSVGInlineText::computeNewScaledFontForStyle(const RenderObject& rende
 {
     // Alter font-size to the right on-screen value to avoid scaling the glyphs themselves, except when GeometricPrecision is specified
     scalingFactor = SVGRenderingContext::calculateScreenFontSizeScalingFactor(renderer);
-    if (!scalingFactor || style.fontDescription().textRenderingMode() == TextRenderingMode::GeometricPrecision) {
+    if (scalingFactor == 1 || !scalingFactor) {
```

Comment:
Can scalingFactor be zero? It is calculated from calculateTransformationToOutermostCoordinateSystem() and if the scaling components of this CTM it is zero, then we will not display anything. So why we do care about correcting scalingFactor here?